packages/neptune-extension/src/common/components/Modal.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactModal from 'react-modal';
+
+import usePlatformModal from 'platform/hooks/usePlatformModal';
+import './Modal.less';
+
+interface ModalProps {
+  onClose: () => void
+}
+
+const Modal: React.FC<ModalProps> = ({
+  onClose,
+  ...props
```

Comment:
⚠️ 

You won't be able to use any other prop anyway because component defines props as `ModalProps`.

I suggest you to change line 11 to this:
`const Modal: React.FC<ModalProps & Omit<ReactModal.Props, 'isOpen'>> = ({`

Explanation:
`ReactModal` exposes `Props` interface, so we use so called Intersection Type ( `&` operator) and provide `ReactModal.Props` interface. The last thing is to exclude required `isOpen` prop because we implement it manually.